docs/Plugins/pagerduty.md

```diff
@@ -14,6 +12,9 @@ This plugin collects all incidents from PagerDuty, and uses them to compute inci
 * [Incident Age](/Metrics/IncidentAge.md)
 * [Incident Count Per 1k Lines of Code](/Metrics/IncidentCountPer1kLinesOfCode.md)
 
+## Supported Versions
+Available for Cloud
```

Comment:
Available for PagerDuty Cloud. Check [this doc](relative url of the supported data sources) for more details.

Reply:
Thanks @Startrekzky but I cannot find the relative url for PagerDuty. Let me know if we can create an issue.

<img width="1120" alt="Screenshot 2023-07-20 at 4 48 11 PM" src="https://github.com/apache/incubator-devlake-website/assets/31725457/8b17aa70-d11b-4b77-a4fa-3920e5fbdf07">